styled-components/src/Text.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components'
+import tag from 'clean-tag'
+
+import {
+  color,
+  fontFamily,
+  fontSize,
+  fontWeight,
+  letterSpacing,
+  lineHeight,
+  textAlign,
+  propTypes
+} from 'styled-system'
+
+const Text = styled(tag.p)(
+  [],
+  color,
+  fontFamily,
+  fontSize,
+  fontWeight,
+  letterSpacing,
+  lineHeight,
+  textAlign
+)
+
+Text.displayName = 'Text'
+Text.propTypes = {
+  ...propTypes.fontFamily,
```

Comment:
With v2, I think the preferred use of propTypes is using `...fontFamily.propTypes` instead of importing the whole `propTypes` object